### PR TITLE
erp outputs endpoint

### PR DESCRIPTION
**Pull request by hdunham**
### Please check if the PR fulfills these requirements
- [x] CHANGELOG.md is updated
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) _will edit wiki once merged into main_

### What kind of change does this PR introduce?
feature

### What is the current behavior?
no erp/inputs (same as erp/help) or erp/outputs urls in the API

### What is the new behavior (if this is a feature change)?
erp/inputs (same as erp/help) or erp/outputs urls added

### Does this PR introduce a breaking change?
no

### Other information:
once merged, update wiki adding 
Get descriptions of possible ERP inputs: /erp/inputs?
Get descriptions of possible ERP outputs: /erp/outputs?
here:
https://github.com/NREL/REopt_API/wiki/1.-Structure-of-the-REopt-API#api-endpoints

### CODE REVIEW

**Comment by Bill-Becker** [comment]
@hdunham I tried to spin the API up locally and test the endpoints, but I'm getting a "connection refused" error. The same GET request to the staging and production servers got a good response for the existing /help endpoint. You've tested this locally?

**Comment by hdunham** [comment]
> @hdunham I tried to spin the API up locally and test the endpoints, but I'm getting a "connection refused" error. The same GET request to the staging and production servers got a good response for the existing /help endpoint. You've tested this locally?

That's weird, yeah I did test it locally. Let me look again.

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,12 @@ Classify the change according to the following categories:
     ##### Removed
     ### Patches
 
+## Develop
+### Minor Updates
+#### Added
+- `/erp/inputs` endpoint (calls `erp_help()`, same as `/erp/help`)
```
Comment (adfarth) [inline CHANGELOG.md:32]: @hdunham is it typical to have two endpoints return the same thing? And is that a correct understanding, that erp/inputs and erp/help will both return the input fields (not the actual user input values) for the ERP function?

File: resilience_stats/views.py [hunk 2]
```diff
@@ -121,6 +126,19 @@ def erp_help(request):
...
+    Served at host/erp/outputs
+    :return: JSON response with all erp outputs
+    """
+
+    try:
+        d = ERPOutputs.info_dict(ERPOutputs)
+        return JsonResponse(d)
+
+    except Exception as e:
+        return JsonResponse({"Error": "Unexpected error in ERP help endpoint: {}".format(e.args[0])}, status=500)
```
Comment (adfarth) [inline resilience_stats/views.py:140]: Should this say "ERP outputs endpoint"?

File: resilience_stats/views.py [hunk 3]
```diff
@@ -105,6 +105,11 @@ def erp_results(request, run_uuid):
         return JsonResponse(resp, status=500)
 
 def erp_help(request):
+    """
+    Served at host/erp/help
```
Comment (adfarth) [inline resilience_stats/views.py:109]: Should this be "Served at host/erp/help and host/erp/inputs" ?

File: CHANGELOG.md (hunk 1 above)
Comment (hdunham) [reply CHANGELOG.md:32]: Yeah that's what we already do for the regular /help and job/inputs endpoints. At some point we could change help to return the info for both inputs and outputs, but for this I wanted a quick solution so I just followed the existing pattern.

Comment (adfarth) [reply CHANGELOG.md:32]: ok cool, I haven't worked with the endpoints much so was curious

**Comment by adfarth** [comment]
> > @hdunham I tried to spin the API up locally and test the endpoints, but I'm getting a "connection refused" error. The same GET request to the staging and production servers got a good response for the existing /help endpoint. You've tested this locally?
> 
> That's weird, yeah I did test it locally. Let me look again.

Oh I guess checking if this ended up working for you when you looked at it again?

File: resilience_stats/views.py (hunk 2 above)
Comment (hdunham) [reply resilience_stats/views.py]: yes, fixed

**Comment by hdunham** [comment]
> > > @hdunham I tried to spin the API up locally and test the endpoints, but I'm getting a "connection refused" error. The same GET request to the staging and production servers got a good response for the existing /help endpoint. You've tested this locally?
> > 
> > 
> > That's weird, yeah I did test it locally. Let me look again.
> 
> Oh I guess checking if this ended up working for you when you looked at it again?

Yeah I've tested it many times and it's always worked

**Review by adfarth — APPROVED** [review]
@hdunham approving, but looks like you may need to merge develop one more time to resolve merge conflicts

**Pull request merged**